### ISSUE TITLE
[java] JUnitTestsShouldIncludeAssert does not detect assertThat() when saving in variable

### ISSUE THREAD

**Issue by marcelmore** [pmd/pmd#4765]
**Affects PMD Version:** 6.55.0

**Rule:**
[JUnitTestsShouldIncludeAssert](https://docs.pmd-code.org/latest/pmd_rules_java_bestpractices.html#junittestsshouldincludeassert)

**Description:**
With AssertJ you can create assert chains in fluent style. It can happen that you save a part in a variable for reuse.
The assertThat() is not detected and the rule is triggered.

**Code Sample demonstrating the issue:**

```java
import static org.assertj.core.api.Assertions.assertThat;

final var dataArea = assertThat(jaxb).isNotNull()
        .extracting(JAXBElement::getValue)
        .isNotNull();
```

**Expected outcome:**

PMD should detect the assertThat. When you remove the variable, the assertThat is detected.

**Running PMD through:**
Gradle

**Comment by adangel** [comment]
This is no longer reproducible under PMD 7.0.0
